doc/query.md

```diff
@@ -76,6 +81,54 @@ if (name == null) {
 
 If an index needs to be changed, first delete the existing index by calling `deleteIndexNamed(String indexName)` where the argument is the index name, then call  `ensureIndexed(List<Object> fieldNames, String indexName)` with the new definition.
 
+#### Managing a Text Index
+
+In order to perform text searches on documents in a datastore a text index needs to be created.  Our text indexing process leverages SQLite's full text search (FTS) functionality, therefore SQLite FTS needs to be available for a text index to be created.
```

Comment:
I think noting which versions we've tested and which have it enabled would be very useful. Although, I couldn't find this after a search...